.github/workflows/e2e-online-upgrade.yml

```diff
@@ -0,0 +1,40 @@
+name: e2e (online-upgrade) tests
+
+on: 
+  push:
+     branches: [main]
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      parallel:
+        description: 'Maximum number of tests to run at once'
+        required: false
+
+jobs:
+
+  e2e-online-upgrade:
+
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Set up e2e environment
+      uses: ./.github/actions/setup-e2e
+      
+    - name: Run e2e tests
+      run: |
+        # Parallelism must be 1.  The testcases will uninstall and install CRDs,
```

Comment:
We don't need to set the parallelism to 1.  That was true only for the operator upgrade tests.  We should set it based on the github input (if present).

Reply:
Makes sense